### ISSUE TITLE
[Feature Request] Render Markdown text realtime 

### ISSUE THREAD

**Issue by Coderx7** [microsoft/vscode-jupyter#1340]
# Feature: Notebook Editor, Interactive Window, Python Editor cells

TLDR:
I suggested, there be a button (maybe next to **Run Cell| Debug Cell |** etc like view(render markdown)
that when clicked would render the whole markdown text in-place, that is the editor accepts the raw markdown like normal and as we type it instantly renders the text for example after a space was given or when a valid markdown code is entered completely! 

## Description
Since Notebooks nearly always are accompanied with descriptive text/explanations/ or basically bodies of text which are normally in Markdown, it would be a great feat to have all these contents rendered inside the editor's cell as well.  
Jupyter notebook provides a similar extension, where, when you are typing, it displays the rendered version of the markdown in realtime. 
For VsCode, however, if we could have all the markdown rendered in realtime, that would be great. No more, we have to run each cell, and see how a specific cryptic tex formula looks like. 
Headings, text in bold, etc and other formattings will stand out instantly and life becomes much easier. 
if we could add some basic rendering for inline markdown in realtime, it would be a very welcomed and very much appreciated feature.  

Thank you all in advance
